core/src/components/segment/test/a11y/e2e.ts

```diff
@@ -11,7 +11,9 @@ test('segment: axe', async () => {
     url: '/src/components/segment/test/a11y?ionic:_testing=true'
   });
 
-  const results = await new AxePuppeteer(page).analyze();
+  const results = await new AxePuppeteer(page)
```

Comment:
Non-blocking/IDK what the Framework team does here - may help to put a TODO that's tied to a JIRA ticket to re-enable the rules here:
```suggestion
  // TODO(Framework-999): Re-enable these rules
  const results = await new AxePuppeteer(page)
```